src/binary/raw_binary_reader.rs

```diff
@@ -663,96 +662,7 @@ impl<R: IonDataSource> RawReader for RawBinaryReader<R> {
             builder.build_at_unknown_offset()
         }?;
 
-        Ok(Some(timestamp))
-    }
-
-    // This method will return year-, month-, and day-precision timestamps as UTC DateTimes
-    // with smaller time units (hour, minute, etc) zeroed out. Longer term, this method
-    // will be replaced by a `read_timestamp` method that returns a proper Timestamp.
-    // TODO: https://github.com/amzn/ion-rust/issues/308
-    fn read_datetime(&mut self) -> IonResult<Option<DateTime<FixedOffset>>> {
```

Comment:
PR tour: the read/write methods for the external `DateTime` type were deprecated in v0.6.1 in favor of the `Timestamp` type.